desk

```diff
@@ -64,7 +64,7 @@ cmd_init() {
         zsh)    local SHELLRC="${HOME}/.zshrc" ;;
     esac
 
-    read -p "Where's your shell rc file? (default: ${SHELLRC}): " \
+    read -r -p "Where's your shell rc file? (default: ${SHELLRC}): " \
```

Comment:
Wow, my bad, that was not supposed to make it in there. I was playing around with the `desk init` command in the terminal and I forgot to remove that.

Thanks for spotting that. I'll take it out.